internal/app/heureka.go

```diff
@@ -69,6 +69,26 @@ func NewHeurekaApp(db database.Database) *HeurekaApp {
 	}
 }
 
+func (h *HeurekaApp) SubscribeHandlers() {
+	// Event handlers for ComponentInstance
+	h.eventRegistry.RegisterEventHandler(
```

Comment:
Isnt that the Issue Matching feature?

Reply:
Same as [here](https://github.com/cloudoperators/heureka/pull/209#discussion_r1762536414)